### PR TITLE
MixtureDensityNetworkの選択確率分布の修正

### PR DESCRIPTION
**Pull request by Geson-anko**
## 概要

#138 

## Submit前の確認項目

- [x] タイトルは一目でわかるようにし、説明文は PR を簡潔に説明するようにしましたか?
- [x] あなたの PR が、異なる変更を束ねたものではなく、ひとつのことだけを行うものであることを確認しましたか?
- [x] この PR で導入されたすべての変更点をリストアップしましたか?
- [x] PR を`make run`コマンドでローカルでテストしましたか?

### CODE REVIEW

**Review by myxyy — COMMENTED** [review]
`NormalMixture.__init__()`の説明`logits, mu, sigma are must be same shape.`あたり事情が変わったようなので直してください

**Comment by Geson-anko** [comment]
@myxyy 修正しました
https://github.com/MLShukai/ami-vconf24/pull/139/commits/6027ddc61c42f177d40b8e94f19370ef62ac2a92

**Review by myxyy — APPROVED** [review]
Approveします
